---
title: "Methylation Analysis"
date: '2025-11-11'
categories:
  - methylation
execute:
  echo: false
  warning: false
  message: false
---

-   Today's mission was to get the 'simple' version of the methylation work up and running on Raven. Last week, I built my bioinformatics environment on my local computer so that I could run the initial QC, trimming and alignment on 1-2 sequences so I could understand what I was doing.
    -   I was able to take the code from Steven's [Mytilus Project repo,](https://github.com/RobertsLab/project-mytilus-methylation/tree/main) strip it down to the barest of essentials (so I could understand it) and run through the Bismark alignment. I was able to do this with one sample (read 1 and 2), but my computer didn't have enough storage/ memory to do the second. Looking at the QC reports, I understand what I see, but I do not understand the Bismark output. This will be a topic in our 1v1 this week.
-   I finally got my repo up on Raven, cleared the cache on my GH Desktop app so that I could actually work with the remote repo, and made script adjustments so that I could call the programs like FastQC, MultiQC, Bismark and Bowtie outside of my locally- created container.
    -   First - and possibly most important, if you update your gitignore file after the first commit without clearing the cache of your GH Desktop app, you will commit/ push files that are listed in the gitignore resulting in a never ending compressing/ writing/ failing loop that will prevent you from doing anything. Thank you StackOverflow for the fix.
    -   Second - I will 100% have to move up from Raven to Klone for this work and beyond. Just running the QC work on Raven has taken hours - will most likely take overnight for the process to finish. Not feeling great about it, but it is a tool that must be used.
